package.json

```diff
@@ -13,6 +13,8 @@
     "prettify": "yarn prettier \"packages/**/*.{js,ts,tsx}\" --write"
   },
   "devDependencies": {
+     "react-hot-loader":"4.8.4",
+    "@hot-loader/react-dom":"16.8.6",
```

Comment:
these two should be added using `yarn`:
`yarn add react-hot-loader @hot-loader/react-dom --dev -W`

this way:
- they will be added with a caret (`^`), as this repo respects semver.
- the yarn.lock will be updated.
- they will be sorted alphabetically for readability.